text/0001-rfc_process.md

```diff
@@ -0,0 +1,244 @@
+- Feature Name: rfc_process
+- Start Date: 2018-08-13
+- RFC PR: #1
+- Issue: N/A
+
+# Summary
+[summary]: #summary
+
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for charting the roadmap of qri. We've seen a number of 
+projects in the distributed space suffer from under-considered design choices 
+and unclear roadmapping, We're hoping strong adherence to a lightweight RFC 
+process can help mitigate these problems. Anoyone should be able get a sense of 
```

Comment:
"Anoyone should be able get a sense of" -> "Anyone should be able get a sense of "